src/server/network/protocol/protocolgame.cpp

```diff
@@ -210,26 +211,26 @@ void ProtocolGame::AddItem(NetworkMessage &msg, const Item* item) {
 			}
 
 			if (lootFlags != 0) {
-				msg.addByte(0x01);
+				containerType = 1;
+				msg.addByte(containerType);
```

Comment:
This is a flag from what I saw, it can be 3 (1 from container and 2 from quiver) and it will expect 4 bytes from the loot flags and 4 bytes for the quiver count

Reply:
I know, this byte now is an enum since, before this update, every time an item with container flag was added into the game, one of the two was 0x01 (true) and the other one was always 0x00 (fqlse), because there is no quiver with loot container flag at the time as far as I know. So they separate it into enums to implement the new feature for the fresh body effect and to short the message buffer as possible. (Sometimes they try haha)

With all that pointed for you and my reply, this is not the PR for features implementation, only for protocol update with the necessary changes to maintain the stability of the client.

The 13.20 features including this one, bosstiary tracker, wheel of destiny changes and so on will come on future PR